alf/utils/value_ops.py

```diff
@@ -66,7 +66,14 @@ def action_importance_ratio(action_distribution, collect_action_distribution,
 
     action_log_prob = dist_utils.compute_log_probability(
         current_policy_distribution, action)
+
     if log_prob_clipping > 0.0:
+        # here we need to clamp the sample_action_log_probs as it is computed
+        # using actions that not sampled from the collect_action_distribution
+        # itself but from another distribution and is more prone to have
+        # numerical issues
```

Comment:
For PPO, `action` is sampled from `collect_dictribution`.

Reply:
> For PPO, `action` is sampled from `collect_dictribution`.

Right, my comment was incorrect. Will update it. Working on an alternative solution by computing the log_prob while sampling.